crates/evm/traces/src/identifier/signatures.rs

```diff
@@ -98,32 +99,36 @@ impl SignaturesIdentifier {
         identifiers: impl IntoIterator<Item = impl AsRef<[u8]>>,
         get_type: impl Fn(&str) -> eyre::Result<T>,
     ) -> Vec<Option<T>> {
-        let cache = match selector_type {
-            SelectorType::Function => &mut self.cached.functions,
-            SelectorType::Event => &mut self.cached.events,
+        let (cache, with_openchain) = match selector_type {
+            SelectorType::Function => (&mut self.cached.functions, true),
+            SelectorType::Event => (&mut self.cached.events, true),
+            // Openchain API does not support custom errors.
```

Comment:
It does, it's in the same format as functions

Reply:
oh, was looking at their API and seen fns and events so I assumed errors not supported, thanks for flagging. Added support and a test with https://github.com/foundry-rs/foundry/pull/9428/commits/35f5dc101c3450773423d54c43be0c24a6817d4e please check